qiskit/aqua/utils/arithmetic.py

```diff
@@ -95,3 +95,33 @@ def next_power_of_2_base(n):
         base += 1
 
     return base
+
+
+def transpositions(permutation):
```

Comment:
Could we have the typehint on the parameter here and Args in docstring describing it. Thx.